stats.rst

```diff
@@ -4,14 +4,89 @@
 Stats Quick Start
 =================
 
-A quick guide to getting the internal stats collection working in flux.
-This guide assumes you have already built a flux binary; if you have not see :ref:`quickstart`.
+Flux includes an optional instrumentation facility to collect internal metrics, or "stats",
+and send them to an aggregator for eventual display in a front end graphing tool such as Graphite.
+The internal stats collection in flux is built on the `statsd <https://github.com/statsd/statsd>`_ protocol and
+following the typing convention used by `brubeck <https://github.com/lukepalmer/brubeck>`_.
+Flux stats supports three metric types: counters, gauges, and timings.
```

Comment:
s/following/follows/

And suggest you for the metric types, you drop the plural, e.g. counter, gauge, and timing.